apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx

```diff
@@ -269,16 +270,16 @@ const SwapForm = () => {
   // Track errors
   useEffect(
     () => {
-      swapError &&
-        trackSwapError(swapError, {
+      (swapError || swapWarning) &&
+        trackSwapError(swapError! || swapWarning!, {
```

Comment:
It could be good as Amine if we want to add the severity level here

Reply:
Thank you, will do! 👍